src/main/webapp/app/controllers/submission/adminSubmissionViewController.js

```diff
@@ -332,9 +332,15 @@ vireo.controller("AdminSubmissionViewController", function ($anchorScroll, $cont
             return $scope.submission.actionLogs[index].entry;
         };
 
-        // TODO: determine how this is set
         var isUmiRelease = function() {
-            return 'no';
+            var umiRelease = 'no';
+            var umiReleaseFilterFieldValue = $scope.submission.fieldValues.filter(function (fv) {
+                if(fv.fieldPredicate.value === "umi_publication") {
+                    umiRelease = fv.value === 'true' ? 'yes' : 'no';
+                    break;
```

Comment:
oh boy. cant break from filter. sorry, my bad. should consider using a for loop to avoid iterating over all field values. please remove break and we can merge